src/Atmos/Model/turbulence.jl

```diff
@@ -517,7 +517,7 @@ function turbulence_tensors(
     Richardson = diffusive.turbulence.N² / (normS^2 + eps(normS))
     f_b² = sqrt(clamp(1 - Richardson * _inv_Pr_turb, 0, 1))
 
-    β = f_b² * (aux.turbulence.Δ)^2 * (α' * α)
+    β = (aux.turbulence.Δ)^2 * (α' * α)
```

Comment:
This change should be documented in the PR or have a commit just for it.

Reply:
@smarras79 can split the commit if he chooses but the notes have been updated.